code/modules/mob/living/life.dm

```diff
@@ -201,6 +204,16 @@
 			AdjustEarDamage(-0.05)
 			AdjustEarDeaf(-1)
 
+/mob/living/proc/handle_stasis()
```

Comment:
I don't think it matters for simple animals, Then again I might enjoy putting a mulebot or Beepsky inside a bag